asciidoctorj-core/src/main/java/org/asciidoctor/extension/Processor.java

```diff
@@ -36,6 +37,10 @@ public RubyHash setConfig(Map<String, Object> config) {
         return null;
     }
 
+    public Block createBlock(AbstractBlock parent, String context, String content, Map<String, Object> attributes) {
```

Comment:
Overloaded this method to not need options as the parameter is optional in the Ruby counterpart